tests/test_docker_compose_state_persistence.py

```
@@ -17,23 +16,23 @@
-        """Clean up test environment"""
+        """Clean up test environment."""
         if self.compose_file:
             self.stop_qdrant_service(self.compose_file, self.temp_dir)
         if self.temp_dir:
             try:
                 shutil.rmtree(self.temp_dir)
-            except Exception as e:
+            except Exception:
                 # Do not fail tests on cleanup, but surface the issue for debugging
-                print(f"Warning: failed to remove temp dir {self.temp_dir}: {e}")
+                pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)